.github/workflows/publish-autoinstrumentation-apache-httpd.yaml

```diff
@@ -60,7 +60,7 @@ jobs:
         uses: docker/build-push-action@v4
         with:
           context: autoinstrumentation/apache-httpd
-          platforms: linux/amd64,linux/arm64
+          platforms: linux/amd64
```

Comment:
This is a breaking change I believe.  Should `linux/arm64` never have been added?